presto-main/src/test/java/com/facebook/presto/sql/gen/BenchmarkPageProcessor.java

```diff
@@ -274,6 +275,7 @@ public static void main(String[] args)
         Options options = new OptionsBuilder()
                 .verbosity(VerboseMode.NORMAL)
                 .include(".*" + BenchmarkPageProcessor.class.getSimpleName() + ".*")
+                .addProfiler(GCProfiler.class)
```

Comment:
What does this do?

Reply:
> What does this do?

This will run the GC profiler. This is why you see the GC stats in the benchmark results:
```
Benchmark                                                         (projectionDataType)   Mode  Cnt       Score        Error   Units
BenchmarkPageProcessor.compiled                                                 BIGINT  thrpt   10    8374.449 ±   1023.859   ops/s
BenchmarkPageProcessor.compiled:·gc.alloc.rate                                  BIGINT  thrpt   10    1451.783 ±    179.079  MB/sec
BenchmarkPageProcessor.compiled:·gc.alloc.rate.norm                             BIGINT  thrpt   10  363640.395 ±      1.385    B/op
BenchmarkPageProcessor.compiled:·gc.churn.PS_Eden_Space                         BIGINT  thrpt   10    1448.711 ±   1904.852  MB/sec
BenchmarkPageProcessor.compiled:·gc.churn.PS_Eden_Space.norm                    BIGINT  thrpt   10  368448.852 ± 487748.838    B/op
BenchmarkPageProcessor.compiled:·gc.churn.PS_Survivor_Space                     BIGINT  thrpt   10       0.019 ±      0.054  MB/sec
BenchmarkPageProcessor.compiled:·gc.churn.PS_Survivor_Space.norm                BIGINT  thrpt   10       4.904 ±     13.838    B/op
BenchmarkPageProcessor.compiled:·gc.count                                       BIGINT  thrpt   10       6.000               counts
BenchmarkPageProcessor.compiled:·gc.time                                        BIGINT  thrpt   10      27.000                   ms
BenchmarkPageProcessor.compiled                                                 DOUBLE  thrpt   10    8805.494 ±    916.612   ops/s
BenchmarkPageProcessor.compiled:·gc.alloc.rate                                  DOUBLE  thrpt   10    1527.888 ±    159.617  MB/sec
BenchmarkPageProcessor.compiled:·gc.alloc.rate.norm                             DOUBLE  thrpt   10  363640.100 ±      0.011    B/op
BenchmarkPageProcessor.compiled:·gc.churn.PS_Eden_Space                         DOUBLE  thrpt   10    1716.294 ±   1812.190  MB/sec
BenchmarkPageProcessor.compiled:·gc.churn.PS_Eden_Space.norm                    DOUBLE  thrpt   10  414368.217 ± 436717.530    B/op
BenchmarkPageProcessor.compiled:·gc.churn.PS_Survivor_Space                     DOUBLE  thrpt   10       0.008 ±      0.023  MB/sec
BenchmarkPageProcessor.compiled:·gc.churn.PS_Survivor_Space.norm                DOUBLE  thrpt   10       1.942 ±      5.771    B/op
BenchmarkPageProcessor.compiled:·gc.count                                       DOUBLE  thrpt   10       7.000               counts
BenchmarkPageProcessor.compiled:·gc.time                                        DOUBLE  thrpt   10      30.000                   ms
```